cmd/azmount/main.go

```diff
@@ -53,93 +54,96 @@ func main() {
 
 	flag.Parse()
 
+	if *logFile != "" {
+		// If the file doesn't exist, create it. If it exists, append to it.
+		file, err := os.OpenFile(*logFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+		if err != nil {
+			logrus.Fatal(err)
+		}
+		defer file.Close()
+		multi := io.MultiWriter(file, os.Stderr)
```

Comment:
Same comment as before.

Reply:
Also, be careful with multiwriter.  Each write happens serially in the order listed.  A slow file write will block all other writes.  An error to one output will cause the log to not be written to any later output.